stablehlo/transforms/Passes.h

```diff
@@ -102,6 +111,24 @@ void populateStablehloCompatibilityExpanderPatterns(
 std::unique_ptr<OperationPass<ModuleOp>> createStablehloRefineArgumentsPass(
     TypeRange refinedTypes);
 
+/// Creates a pass that wraps StableHLO ops in CompositeOp.
+/// The pass takes in a map of op type to attribute predicate. The attribute
+/// predicate is a function that takes in an operation, of the given op type,
+/// and returns a list of attributes to be added to the CompositeOp.
+using AttributePredicate =
```

Comment:
The name `AttributePredicate` might be slightly misleading since it doesn't just check attributes but instead transforms them into a `NamedAttrList` for the `CompositeOp`. Perhaps `CompositeAttributeProvider` and  `CompositeAttributeProviderMap` to be more descriptive?

Reply:
Nice suggestion. Will add.